fedmsg_meta_fedora_infrastructure/coprs.py

```diff
@@ -17,9 +17,32 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 
+import copy
+
 from fedmsg_meta_fedora_infrastructure import BaseProcessor
 from fasshim import gravatar_url
 
+_statuses = {
+    0: 'failed',
+    1: 'success',
+    3: 'running',
+    5: 'skipped',
+}
+
+
+_long_template = """Package:  {pkg}
+COPR:     {owner}/{copr}
+Built by: {user}
+Status:   {status}
+ID:       {build}
+
+Logs:
+  Build:     http://copr-be.cloud.fedoraproject.org/results/{owner}/{copr}/{chroot}/build-{build}.log
```

Comment:
These build logs are kinda useless, they are copr backend logs. The actual build log (make output and stuff) are in

http://copr-be.cloud.fedoraproject.org/results/{owner}/{copr}/{chroot}/{pkg}/build.log